libexec/bats

```diff
@@ -55,7 +60,7 @@ expand_path() {
 BATS_LIBEXEC="$(abs_dirname "$0")"
 export BATS_PREFIX="$(abs_dirname "$BATS_LIBEXEC")"
 export BATS_CWD="$(abs_dirname .)"
-export PATH="$BATS_LIBEXEC:$PATH"
+export PATH="$BATS_LIBEXEC:$(getconf PATH 2>/dev/null):$PATH"
```

Comment:
Ugh. Apologies, the >/dev/null in the sed in libexec/bats-exec-test hid the issue. /usr/xpg4/bin/sed is required for the expression to work